ui/src/app/base/components/StatusBar/StatusBar.tsx

```diff
@@ -34,17 +37,48 @@ const getLastCommissionedString = (machine: MachineDetails) => {
   }
 };
 
+const getSyncStatusString = (syncStatus: string) => {
+  if (syncStatus === "") {
+    return "Never";
+  }
```

Comment:
Do we need an intermediate string here while the machine is deploying, like with commissioning? Having it display "Never" for next sync makes it seem like something's gone wrong

Reply:
Good spot! You actually made me realise there's a fault in my logic here. We only ever want to display sync status if the machine is deployed. Updated this.

Reply:
Will try to discuss and make decisions how to handle potential edge cases in the spec file.